pygef/gef.py

```diff
@@ -619,11 +624,15 @@ def calculate_friction_number(df):
         return df
 
     @staticmethod
-    def calculate_elevation_with_respect_to_nap(df, zid, height_system):
+    def calculate_elevation_with_respect_to_nap(zid, height_system):
         if zid is not None and height_system == 31000:
-            df["elevation_with_respect_to_NAP"] = np.subtract(zid, df["depth"])
+            return (
+                pl.col("depth")
```

Comment:
No need for a lambda:

```python
(zid - col("depth")).alias("...")
```

Reply:
I'm getting this error:

```File "/home/runner/work/pygef/pygef/pygef/gef.py", line 652, in calculate_elevation_with_respect_to_nap
    return (zid - pl.col("depth")).alias("elevation_with_respect_to_nap")
TypeError: unsupported operand type(s) for -: 'float' and 'Expr'```

Which you can also see in the CI.

Reply:
Oh right, the subtraction overloading goes to the left hand operand. In that case:

```python
(pl.lit(zid) - col("depth")).alias("...")
```

We explicitly create a literal expression.